DependencyInjection/Configuration.php

```diff
@@ -51,7 +51,7 @@ public function getConfigTreeBuilder()
                                         if (!isset($features[$featureName])) {
                                             throw new InvalidConfigurationException("Uknown feature: {$featureName}");
                                         }
-                                        $featureMask += isset($features[$featureName]) ? $features[$featureName] : 0;
+                                        $featureMask &= $features[$featureName];
```

Comment:
@lisachenko Please note this one it is important.

isset() is not required here, you are already checking it above.

Mask should be built with bitwise and operator, not + operator, right?